### PR TITLE
Feature see courses

### PR DESCRIPTION
**Pull request by meganokane97**

### CODE REVIEW

File: seeCourses.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const Database = require('./db.js');
+
+console.log("here")
+
+function seeCourses(){
+    console.log("get data")
+    
+    Database.query("SELECT * FROM Course").then(rows =>{
```
Comment (rorysproule1) [inline seeCourses.js]: Is this js file still needed?

Comment (meganokane97) [reply seeCourses.js]: no

File: seeCourses.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Kainos Map</title>
```
Comment (ghost) [inline seeCourses.html:6]: Map should be in capitals.

**Comment by ghost** [comment]
The branch name doesn't match the required format.

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**